### PR TITLE
docs: update docutils

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

Related to #874 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,11 +4,13 @@
 Added:
 ^^^^^^
 * Added the readthedocs page for BALSAMIC variant-calling filters #867
+* Project requirements (setup.py) to build the docs
```
Comment (ashwini06) [inline CHANGELOG.rst:7]:
```suggestion
* Project requirements (setup.py) to build the docs #874 
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,11 +4,13 @@
...
 * Generate cram from umi-consensus called bam files #865
 
 Changed:
 ^^^^^^^^
 * Updated the bioinfo tools version numbers in BALSAMIC readthedocs #867
+* Sphinx version fixed to <0.18
```
Comment (ashwini06) [inline CHANGELOG.rst]:
```suggestion
* Sphinx version fixed to <0.18 #874 
```

**Review by ashwini06 — COMMENTED** [review]
Great job @ivadym for fixing this 💯

**Review by ivadym — APPROVED** [review]
👍

**Pull request merged**